.github/pull_request_template.md

```diff
@@ -0,0 +1,62 @@
+# Which issue does this PR close?
```

Comment:
If we start to use this, that will get duplicated with the `* Closes ..` that the bot adds. Do we then want to remove the bot action? (or some way to unify those? but since this is free form, it might be difficult to have the bot check it)

Reply:
Remove this line?

Reply:
That's also an option. As long as we still require an issue and ask it to be in the title, then this is indeed a bit redundant (and we could have a hidden comment explaining that it should be in the title)